test/Formatter/SimpleFormatterTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * @author     Moritz Reiter
+ * @category   Horde
+ * @package    Log
+ * @subpackage UnitTests
+ */
+
+namespace Horde\Log\Test\Formatter;
+use PHPUnit\Framework\Testcase;
+use Horde\Log\Formatter\SimpleFormatter;
+use Horde\Log\LogMessage;
+use Horde\Log\LogFormatter;
+
+
+class SimpleFormatterTest extends TestCase
+{
+        public function setup(): void
+        {
+            $this->filter = new SimpleFormatter();
+        }
+
+        public function Formatisnull()
+        {
+            $this->assertTrue($this->filter->accept(array('message' => '', 'level' === 0)));
+        }
+
+        public function Formatisnotnull()
+        {
+            $this->assertFalse($this->filter->accept(array('message' => '', 'level' != 1)));
+        }
```

Comment:
Not sure why these methods are here, but they are not executed by phpunit because they dont have `test` in their name.